src/API/timeseries/timeseries.service.js

```diff
@@ -9,7 +9,7 @@ Promise.promisifyAll(MongoClient);
 const debug = Debugger('SH_BE:new-device-handler');
 const STATUS_EVENT = 'status';
 const STRATEGY = 'averages';
-const periods = ['month', 'day', 'hour', 'minute', 'second'];
+const periods = ['month', 'day', 'hour'];
```

Comment:
"second" property was not used here before, and to display statistic by hour we are using average values by minute and now we decided  not to store this values in the database